modules/misc/news.nix

```diff
@@ -103,7 +103,7 @@ in
       entries = mkOption {
         internal = true;
         type = types.listOf entryModule;
-        default = [];
```

Comment:
What happened here? These changes shouldn't be here.

Reply:
Oops, probably formatting.